docs/configuration/prefill/haal_centraal.rst

```diff
@@ -49,6 +49,55 @@ Configuration
 
 The Haal Centraal configuration is now complete.
 
+Configuration of *protocollering* headers
+=========================================
+
+When using the BRP API via a gateway, you might need to configure extra headers.
+
+*Doelbinding* and *Verwerking*
+------------------------------
+
+These two values correspond to the following headers: ``x-doelbinding`` and ``x-verwerking``.
+They can be configured per form or as a global default.
+
+Configuration per form
+~~~~~~~~~~~~~~~~~~~~~~
+
+1. Create a form with a component using a prefill plugin. An example can be found :ref:`here <example_prefill>`.
+   Be sure to select *Haal Centraal: BRP Personen Bevragen*.
+2. In the *Advanced configuration* tab under the *Processing* group, fill in the values for the two headers:
+
+   * **BRP Personen purpose limitation header value**: The *Doelbinding* value.
+   * **BRP Personen processing header value**: The *Verwerking* value.
```

Comment:
can you add some more information that the values depend on the vendor of the gateway? See the (Dutch) help texts from the release branch (should be on our test env by now too).